src/helpers.ts

```diff
@@ -1,6 +1,16 @@
-import {QuillFormat} from './quill-editor.interfaces'
+import { QuillFormat } from "./quill-editor.interfaces";
```

Comment:
i do not use double quotes and semicolons

Reply:
As I said, my IDE settings messed up everything. If you use VS Code, maybe we can create a workspace settings.json so that anyone using this repo will use only those settings.

Reply:
nope i will not do it:

1. those settings contains many local stuff, like plugins and so
2. i will not include those settings for every editor a contributor uses
3. Things like code formatting and so on should not be done automatically by the editor
4. Those things are linted by eslint and all dependencies are installed automatically. Just use the lint task to get that thing your IDE sadly does automatically... and for every project (which is bad)